packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -114,6 +130,7 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
 
   return (
     <ProtectedPage seo={seo} fallback={<></>} shouldFallback={!user}>
+      {isPopupOpen && <SquadTourPopup onClose={onClosePopup} />}
```

Comment:
Why don't you use the useModal here?

Reply:
There was a long discussion about whether should this be modal or not. The conclusion was to introduce a new popup since the behavior is different. The user should be able to interact with the components outside this popup and keep it open until the user explicitly closes it which our modal doesn't allow due to having an overlay.

Reply:
Oh I see, but it's different from the SquadTour modal?

Reply:
Yes, they will be two different components but using the same content as `SquadTour`.

Reply:
Got it then, makes sense.

Seems to work fine while testing it locally.